content/integrations/awsrds.md

```diff
@@ -74,79 +76,78 @@ There are 3 options for monitoring RDS instances. You can choose to use standard
   17. Click the **Create Function** button.
 
 
-* **Native Database Integration**
+### Native Database Integration
 
   1.  Navigate to the AWS Console and open the RDS section to find the instance you want to monitor.
       ![][1]
   2.  Copy the endpoint URL (e.g. **mysqlrds.blah.us-east1.rds.amazonaws.com:3306**); You will need it when you configure the agent. Also make a note of the `DB Instance identifier` (e.g. **mysqlrds**). You will need it to create graphs and dashboards.
 
-# Configuration
-
-* **Standard RDS Integration**
-
-  1.  Ensure RDS is checked in the AWS Integration tile.
-
-* **Enhanced RDS Integration**
+## Configuration
 
-  1.  Ensure RDS is checked in the AWS Integration tile.
+### Standard RDS Integration
 
-* **Native Database Integration**
+  Ensure RDS is checked in the AWS Integration tile.
```

Comment:
Do not indent. This is what's causing it to be a bullet point, I think.

Same for other sections that indent. On the Docs site, paragraphs should never be indented.